Utility/Utility.Test/UserDataValidationTest.cpp

```diff
@@ -0,0 +1,123 @@
+#include <catch2/catch.hpp>
+
+#include "Utility/UserDataValidation.hpp"
+
+TEST_CASE("Login validation test")
+{
+    SECTION("Input valid login")
+    {
+        std::string rightLogin("some_nick34");
```

Comment:
The same there (Why not a std::string_view I mean)

Reply:
Because there is no overloaded function 'std::regex_match' with 'std::string_view' parameter